pimd/pim_nht.c

```diff
@@ -623,6 +657,18 @@ static int pim_ecmp_nexthop_search(struct pim_instance *pim,
 			continue;
 		}
 
+		if (!pim_ifp->pim_enable) {
```

Comment:
Please add this as an OR condition in the previous check. It's doing the same thing.